gui/components/forms/postRegistration.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useState } from 'react'
+import { TextInput, StyleSheet, TouchableOpacity, Picker } from 'react-native'
+import DropDownPicker from 'react-native-dropdown-picker';
+import { Text, View } from '../Themed'
+import globalStyles from '../../styles/global'
+
+const interestsInit = [{ value: 'hiking', label: "Hiking" }, { value: 'biking', label: "Biking" }, { value: 'swimming', label: "Swimming" }, { value: 'fishing', label: "Fishing" }, { value: 'rock_climbing', label: "Rock Climbing" }, { value: 'trailblazing', label: "Trailblazing" }, { value: 'running/jogging', label: "Running/Jogging" }, { value: 'horse_back_riding', label: "Horse Back Riding" }, { value: 'rafting', label: "Rafting" }, { value: 'camping', label: "Camping" }, { value: 'ziplining', label: "Ziplining" }, { value: 'skiing/snowboarding', label: "Skiing/Snowboarding" }, { value: 'bungee_jumping', label: "Bungee Jumping" }, { value: 'canoeing', label: "Canoeing" }, { value: 'skydiving', label: "Skydiving" }]
+
+const sexInit = [
+  { label: 'Male', value: 'male' },
+  { label: 'Female', value: 'female' },
+]
+
+const oftenInit = [
+  { value: 1, label: 'Less than once a month' },
+  { value: 2, label: 'Once a month' },
+  { value: 3, label: 'Twice a month' },
+  { value: 4, label: 'Weekly' },
+  { value: 5, label: 'More than once a week' },
+]
+
+const experienceInit = [
+  {value:1, label:'Novice'},
+  {value:2, label:'Beginner'},
+  {value:3, label:'Intermediate'},
+  {value:4, label:'Expert'},
+  {value:5, label:'Master'}
+]
```

Comment:
Please get these values from the API to build the dropdown values. 

Activities: api/activity
Attitudes: api/attitudes
Skill levels: api/skillLevel

These are the values that should go into the drop-downs.

Then, POST a message to api/user/interest that looks like: 
```
[ 
  'interests': [
    { 
      'activity': <activity value>
      'attitude': <attitude value>
      'skillLevel': <skill level value>
    }
    ...
]
```